doc/app-flows.md

```diff
@@ -0,0 +1,33 @@
+# App Flows
+
+## Main App Flow
+
+1. User receives a text with a redemption link.
+
+2. Check if the customer is logged in with `Get Current Customer`.
+
+3. Use `Fetch Unclaimed Orders` to get information about the order.
+
+4. Check in your system if a customer API token exists for that customer.
+     - If the token does not exist, follow [New Customers](#new-customers).
+     - If the token does exist, follow [Existing Customers](#existing-customers).
+
+5. Claim the order with the redemption code using `Claim Orders`.
```

Comment:
This could use a note that if the customer is not logged in, an headless customer will be created when the order is claimed, and that customer will be logged in.

Reply:
I didn't realize the SDK will create a headless customer automatically in this case, TIL.